app/uk/gov/hmrc/hec/services/FileCreationService.scala

```diff
@@ -214,9 +220,12 @@ class FileCreationServiceImpl @Inject() (timeProvider: TimeProvider) extends Fil
               hecTaxCheck.createDate.withZoneSameInstant(ZoneId.of("GMT")).format(DATE_TIME_FORMATTER),
             taxCheckCode = hecTaxCheck.taxCheckCode.value,
             taxCheckExpiryDate = hecTaxCheck.expiresAfter.format(DATE_FORMATTER),
-            onlineApplication = if (i.source === Digital) 'Y' else 'N'
+            onlineApplication = if (i.source === Digital) 'Y' else 'N',
+            emailAddress =
+              FeatureEnabledField(hecTaxCheck.latestTaxCheckEmailSentTo.map(_.value), emailAddressFieldEnabled)
```

Comment:
can we store this `hecTaxCheck.latestTaxCheckEmailSentTo.map(_.value), emailAddressFieldEnabled `in a def or val, we are using it more than once

Reply:
:+1: updated [here](https://github.com/hmrc/hec/pull/67/commits/429c29e18f496bd35e9ccee5730b7deee6ba53e4)